webapp/README.md

```diff
@@ -71,11 +71,10 @@ projects:
 
 Start the server with `yarn dev` in `webapp`.
 
-Open `http://localhost:3000/projects` to view the configured projects.
+Open the URL provided in the terminal to view projects.
```

Comment:
what is "the URL provided in terminal"?

Reply:
When I run yarn dev in webapp, next prints
▲ Next.js 13.5.6
  - Local:        http://localhost:3000

The Local URL is what I meant.